.github/actions/openziti-tunnel-build-action/ubuntu-22.04/Dockerfile

```diff
@@ -1,6 +1,11 @@
-FROM ubuntu:jammy
+ARG CMAKE_VERSION="3.22.3"
 
-LABEL org.opencontainers.image.authors="netfoundry.io"
+# upstream of Ubuntu Jammy 22.04 LTS
+FROM debian:bookworm
```

Comment:
Debian Bookworm is the contemporary of Ubuntu Jammy, but there's always the possibility that Bookworm will roll forward to a new GLIBC that is unavailable in Jammy. We have the option now, or later, to switch Jammy's build to the predecessor, Debian Bullseye, which will always have a compatible GLIBC, but may introduce other OS differences.